src/app/components/listing/details/cqms/cqms.jsx

```diff
@@ -65,40 +70,41 @@ function ChplCqms(props) {
                   <TableCell>170.315 (c)(4)</TableCell>
                 </>
               )}
-          </TableRow>
-        </TableHead>
-        <TableBody>
-          { cqms.filter((cqm) => viewAll || cqm.success)
-            .sort(sortCqms)
-            .map((cqm) => (
-              <TableRow key={cqm.id ?? cqm.cmsId}>
-                <TableCell>
-                  <span className="sr-only">{ cqm.success ? 'meets' : 'does not meet' }</span>
-                  { edition.name === '2011' && cqm.success
+            </TableRow>
+          </TableHead>
+          <TableBody>
+            { cqms.filter((cqm) => viewAll || cqm.success)
+              .sort(sortCqms)
+              .map((cqm) => (
+                <TableRow key={cqm.id ?? cqm.cmsId}>
+                  <TableCell>
+                    <span className="sr-only">{ cqm.success ? 'meets' : 'does not meet' }</span>
+                    { edition.name === '2011' && cqm.success
                     && (
                       <CheckIcon fontSize="large" />
                     )}
-                  { cqm.successVersions?.length > 0 && cqm.successVersions.join(', ') }
-                  { !cqm.success
+                    { cqm.successVersions?.length > 0 && cqm.successVersions.join(', ') }
+                    { !cqm.success
                     && (
-                      <CheckBoxOutlineBlankIcon fontSize="large" />
+                      <NotInterestedIcon fontSize="large" />
```

Comment:
Are we sure we want to do this? This isn't consistent with the way unmet criteria are displayed. If we want to stick with the the criteria pattern, we would need to just have this blank, and then somehow style the entire row of the table as "disabled"